Sources/ChessKit/Special Moves/EnPassant.swift

```diff
@@ -14,16 +14,16 @@ struct EnPassant: Equatable, Hashable, Sendable {
         Square(pawn.square.file, pawn.color == .white ? 3 : 6)
     }
 
-    /// Determines whether or not the pawn can be captured by en passant.
+    /// Determines whether or not the pawn could be captured by en passant.
     ///
     /// - parameter capturingPiece: The piece that is capturing the contained pawn.
-    /// - returns: Whether `capturingPiece` can capture `pawn`.
+    /// - returns: Whether `capturingPiece` could capture `pawn`.
     ///
     /// `capturingPiece` must be an opposite color pawn that is on the
     /// same rank as the target pawn and exactly 1 file away from the
     /// target pawn for this method to return `true`, otherwise `false`
     /// is returned.
-    func canBeCaptured(by capturingPiece: Piece) -> Bool {
+    public func couldBeCaptured(by capturingPiece: Piece) -> Bool {
```

Comment:
Does this need to be `public`? `EnPassant` isn't public right now so I don't think this will have any effect.